tpu-client-next/src/metrics.rs

```diff
@@ -0,0 +1,47 @@
+use {
+    crate::SendTransactionStats,
+    solana_metrics::datapoint_info,
+    std::{sync::Arc, time::Duration},
+    tokio::time::interval,
+    tokio_util::sync::CancellationToken,
+};
+
+impl SendTransactionStats {
```

Comment:
What if some hypothetical client will decide to report metrics somewhere else (not influxdb)? Am I right that the idea is that in this case client will just write his own function `report_to_something` (because he cannot do impl for `SendTransactionStats`)?